src/plugins/workspace/public/application.tsx

```diff
@@ -6,19 +6,74 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Router } from 'react-router-dom';
-
 import { AppMountParameters, CoreStart } from '../../../core/public';
-import { OpenSearchDashboardsContextProvider } from '../../../plugins/opensearch_dashboards_react/public';
-import { WorkspaceApp } from './components/workspace_app';
+import { OpenSearchDashboardsContextProvider } from '../../opensearch_dashboards_react/public';
+import { WorkspaceListApp } from './components/workspace_list_app';
+import { WorkspaceCreatorApp } from './components/workspace_creator_app';
+import { WorkspaceUpdaterApp } from './components/workspace_updater_app';
+import { WorkspaceOverviewApp } from './components/workspace_overview_app';
+
+export const renderListApp = (
+  { element, history, appBasePath }: AppMountParameters,
+  services: CoreStart
+) => {
+  ReactDOM.render(
+    <Router history={history}>
```

Comment:
As we are using applications as our router, is the Router component essential for the workspace app to run?